ocrd_all_releases.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
This could be generated with the `release.sh` tool we use in ocrd_all.

Reply:
So we should toss `release.py` and use `release.sh` in ocrd_all instead?

Reply:
See https://github.com/OCR-D/kwalitee-dashboard-back-end/pull/7#discussion_r901283881.